### DISCUSSION TITLE
Feature Request: Fortran Language Support in Zed

### DISCUSSION THREAD

**Discussion by gratitude38** [zed-industries/zed#9065] [Feature Requests]
Hi, I'm excited about Zed and was wondering if there are plans to support **Fortran**, specifically with features like syntax highlighting, code completion, and error detection. Running Fortran code directly would also be great, though it's not urgent. Thanks for considering my request!

**Comment by Moshyfawn** [comment]
There's a community plugin request in the extensions repo you can +👍 zed-industries/extensions#172

**Comment by gratitude38** [comment]
Thanks for pointing to the right direction!
